core/profile.c

```diff
@@ -1156,7 +1155,8 @@ static int calculate_ccr_po2(struct plot_data *entry, const struct divecomputer
 				return sump / 3;
 		}
 	default: // This should not happen
-		assert(np <= 3);
+		// assert(np <= 3);
```

Comment:
It seems reasonable to _not_ crash here. :-)
The question is should we just change the voting logic to work with an arbitrary number of sensors?

Alternatively we should probably return `entry->o2pressure.mbar` here and not 0, which is a bad value for ppO2.

Reply:
Robert is working with Jef to figure out the right behavior

Reply:
I have got an idea:
Keep voting out sensors that deviate more than a set percentage from the average of the rest until none of these are left (or only two sensors are left), then average the remaining sensors.
This seems to be aligned with what Shearwater computers (used as controllers in a number of CCRs) do, and very similar to the behaviour of other CCRs that document their inner workings, like the APD Inspiration.